### PR TITLE
docs: add docs for file imports

### LINKED ISSUE #250: Add docs for supported markdown syntax
### Is your feature request related to a problem?

I don't know which md syntaxes are supported in the markdown that is used by tutorialkit - callouts etc...

### Describe the solution you'd like.

Would be great to have a page with the supported syntaxes - or a link to one

### Describe alternatives you've considered.

.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
**Description**

- Closes #250
- Adds docs for file imports using shortcuts `file` and `solution`

### CODE REVIEW

**Review by AriPerkkio — CHANGES_REQUESTED** [review]
Thanks for the PR! :raised_hands:

This is good a start. For the #250 I think we want to have documentation for all supported markdown syntax. It should have links to [Astro](https://docs.astro.build/en/guides/content/) and [`expressive-code`](https://expressive-code.com/key-features/syntax-highlighting/), and probably some small examples from each. Also the [Callouts](https://tutorialkit.dev/reference/theming/#callouts).

The structure could be something like:

```md
// creating-content.mdx

## Markdown support
...

### Displaying code
<some examples showing how to select language for syntax highlight and highglight specific lines. Add links to asto & expressive-code.>

### Importing files
<the guide this pr already has>

### Callouts
<examples and description of callouts>
```

@Nemikolh & @sulco any thoughts?

**Comment by Nemikolh** [comment]
@AriPerkkio, I agree with your proposed structure! Long term we could even have the `# Markdown support` section describes everything that's possible but right now if it shows the most commonly used things and link to `expressive-code` that would be a great start :raised_hands:

**Comment by kevinzunigacuellar** [comment]
Thanks for the review and clarification. I wasn't sure if I should write an entire markdown/mdx section so I decided to start with just file imports.

File: docs/tutorialkit.dev/src/content/docs/guides/creating-content.mdx [hunk 1]
```diff
@@ -144,3 +144,72 @@ src/templates
...
+![Content](../reference/images/theming-callout.png)
+
+To customize the styles of a callout, check out the [theming reference](/reference/theming/#callouts).
+
+
+### Code blocks
+
+TutorialKit offers a comprehensive set of code block features powered by Expressive Code. It includes all core features, along with optional plugins like collapsible sections and line numbers. For a full overview, check out the [Expressive Code documentation](https://expressive-code.com/).
+
+````txt
```
Comment (AriPerkkio) [inline docs/tutorialkit.dev/src/content/docs/guides/creating-content.mdx:175]:
```suggestion
````md title="content.md"
```

**Review by AriPerkkio — APPROVED** [review]
Looks good to me, thanks @kevinzunigacuellar! 🙌 

In future we can extend the examples to show more features provided by `expressive-code` and Astro, but for now it's good enough to provide links and that one example. 

These changes will be automatically deployed when we release the next version of TutorialKit.

**Pull request merged**